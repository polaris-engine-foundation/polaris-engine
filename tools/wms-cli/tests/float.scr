func main() {
     a = 0.0;
     b = 1.0;
     c = 3.14159265;
     d = a + b + c;
     print(d);
     d = a + b - c;
     print(d);
     d = a + b * c;
     print(d);
     d = a + b / c;
     print(d);
     print("string conversion");
     print("" + d);
}
