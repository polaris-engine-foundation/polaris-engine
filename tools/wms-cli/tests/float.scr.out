4.141593
-2.141593
3.141593
0.318310
string conversion
0.318310
